shell/browser/api/electron_api_utility_process.cc

```diff
@@ -140,6 +143,7 @@ UtilityProcessWrapper::UtilityProcessWrapper(
       int devnull = open("/dev/null", O_WRONLY);
       if (devnull < 0) {
         PLOG(ERROR) << "failed to open /dev/null";
+        close(devnull);
```

Comment:
Since `devnull` isn't a valid fd, wouldn't this `close()` call fail with `EBADF`?

Reply:
it didn't in test but we figured out we don't need this i don't think, not if we have `if (io_handle == IOHandle::STDIN)` above.